handshake/connection_parameters_manager.go

```diff
@@ -61,14 +63,19 @@ var (
 )
 
 // NewConnectionParamatersManager creates a new connection parameters manager
-func NewConnectionParamatersManager(pers protocol.Perspective, v protocol.VersionNumber) ConnectionParametersManager {
+func NewConnectionParamatersManager(
+	pers protocol.Perspective, v protocol.VersionNumber,
+	maxReceiveStreamFlowControlWindow protocol.ByteCount, maxReceiveConnectionFlowControlWindow protocol.ByteCount,
```

Comment:
That will create an import loop, and I don't see how you can resolve that.
We could use the `TransportParameters` struct for that https://github.com/lucas-clemente/quic-go/blob/master/handshake/interface.go#L22-L24.